Team7/src/Motors.java

```diff
@@ -0,0 +1,77 @@
+
+import lejos.hardware.motor.EV3LargeRegulatedMotor;
+import lejos.hardware.port.MotorPort;
+import lejos.utility.Delay;
+
+public class Motors extends Thread {
+  static EV3LargeRegulatedMotor motorA = new EV3LargeRegulatedMotor(MotorPort.A);
+  static EV3LargeRegulatedMotor motorD = new EV3LargeRegulatedMotor(MotorPort.D);
+
+  public EV3LargeRegulatedMotor getMotorA() {
+    return motorA;
+  }
+
+  public EV3LargeRegulatedMotor getMotorD() {
+    return motorD;
+  }
+
+  public static void run1() {
+    motorA.setSpeed(200);
+    motorD.setSpeed(100);
+    motorA.forward();
+    motorD.forward();
+  }
+
+  public static void run2() {
+    motorA.setSpeed(100);
+    motorD.setSpeed(200);
+    motorA.forward();
+    motorD.forward();
+  }
+
+  public static void run3() {
```

Comment:
please write some comments for motors speed.

Reply:
our motors speed is 450 m/h